Documents/Design/ZTag User Wearable Container Diagram

```diff
@@ -0,0 +1,148 @@
+<mxfile host="app.diagrams.net" modified="2023-03-04T21:44:31.820Z" agent="5.0 (Windows)" etag="emzj11SK3La5-8Eu15bw" version="21.0.2" type="github">
```

Comment:
This file does not have the .drawio extension. 
This makes it a pain to open.

Reply:
you open it with draw.io by linkingyour github